src/shared/components/palette/utils/date.pipe.ts

```diff
@@ -67,7 +67,8 @@ export class DatePipe implements PipeTransform {
             return this.transform(shortISO, zone, format);
           }
           // If it did include time, we want a full ISO string.
-          return this.transform(d.toISOString(), zone, format);
+          const thisMoment = moment(d).format('YYYY-MM-DDTHH:mm:ss.SSSZ');
```

Comment:
_Nitpick:_
Did you not previously declare a bunch of constants for date formats? It would be good to use one of those here rather than inline a string constant.

Reply:
Not completely sure what the suggestion was here actually but I have used a constant for the format and placed it at the start of the translate method for use within it